### DISCUSSION TITLE
Visual Studio

### DISCUSSION THREAD

**Discussion by GAMECHANGE** [community/community#60008] [New to GitHub]
### Select Topic Area

General

### Body

Hi everyone! Very nice to connect and I am very excited about this accessibility and softwares capabilities. I am learning at each exciting step. My goal is to create and share a visual gig five which can be shared across social media Ina fun, kind, friendly way.

**Comment by Akash1134** [comment]
Hi @GAMECHANGE , 👋 Welcome to the GitHub Community on Discussions, Happy to have you here 🙂

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).

**Reply by GAMECHANGE** [reply]
THANK YOU
